references/knowledge-base/playback.mdx

```diff
@@ -0,0 +1,99 @@
+<Card title="A warning on the stream's page in the Livepeer Studio UI.">
+  Diagnose: This is due to the low latency playback feature using WebRTC. If
+  there are bframes in the stream, the default behavior is to fallback to HLS
+  playback, which means a slight increase in latency. Workaround: In the
+  settings of the broadcaster, turn off the use of frames. If using OBS, select
+  Livepeer as the service for the video settings.
+  [https://docs.livepeer.org/guides/developing/stream-via-obs#input-your-stream-settings](https://docs.livepeer.org/guides/developing/stream-via-obs#input-your-stream-settings)
+</Card>
+
+<Card title="Streams sometimes have a difficult time reconnecting to the feed and stop.">
+  Diagnose: Reconnection: If a stream is down, attempt to retry for that stream
+  from other nodes nearby until there is a connection(using Livepeer player)
+  Workaround: Implement a retry logic or switch over to the Livepeer player with
+  the SDK or iframe embeded player.
+</Card>
+
+<Card title="Is there a way we could update the viewerId and video watch event to livestream / asset through Livepeer API?">
+Diagnose: We do not support pagination yet but it is a pending feature. You can use the playbackId as a workaround.
+Workaround: A short term workaround could be to "paginate" by playbackId instead.
+- First list all playback ids with: `/api/data/views/query?breakdownBy[]=playbackId`
+- then viewers by playbackId with `/api/data/views/query?playbackId=<pid>&breakdownBy[]=viewerId`
+</Card>
+
+<Card title="Is there a way to update width and height of iframe embeded player">
+Diagnose: Yes, you can use the iframe embeded player and set the width and height in the iframe tag, however the best way to do this is to use the aspect ratio. 
+</Card>
+
+
+<Card title="How to enforce webrtc playback, to ensure it is always low latency otherwise fail?">
+There are two solutions to accomplish this:
+1. If you are using react component, you can pass `lowLatency="force"` to the player component.
+You can
+```js
+function PlayerComponent() {
+  return (
+    <Player
+      title="My Livestream"
+      playbackId="wwl88k4g8f5eese9"
+      lowLatency="force"
+    />
+  );
+}
+```
+
+2. If you are using the iframe embeded player, you can add `lowLatency=true` to the query string.
```

Comment:
This section does not appear to be part of the card, maybe a syntax issue?

Reply:
Actually it is a part of card :)

<img width="783" alt="image" src="https://github.com/livepeer/docs/assets/56798748/eb4a212a-fc4c-4391-aea3-529df57196f5">

Reply:
The code just doesn't show the highlighted colors.

Reply:
Fixed #503